test/chat_api/slack_test.exs

```diff
@@ -160,6 +161,23 @@ defmodule ChatApi.SlackTest do
       assert conversation.assignee_id == primary_user.id
     end
 
+    test "fetch_valid_user/1 reject disabled users and fetch the oldest user.",
+         %{account: account} do
+      {:ok, disabled_user} =
+        account
+        |> user_fixture()
+        |> Users.disable_user()
+
+      primary_user = user_fixture(account)
+
+      # Make sure that secondary_user is inserted later.
+      :timer.sleep(1000)
```

Comment:
can we do 100ms in stead of a full 1s here? just don't want to slow down tests too much 😛 

```suggestion
      :timer.sleep(100)
```